recipes/graylog-illuminate/files/post-install

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+
+# Flag to indicate if this is an upgrade
+upgrade="false"
+
+case "$1" in
+	# DEB based systems
+	configure)
+		if [ -z "$2" ]; then
+			upgrade="false"
+		else
+			upgrade="true"
+		fi
+		;;
+	abort-deconfigure|abort-upgrade|abort-remove)
+		;;
+	# RPM based systems
+	1)
+		# Installation
+		upgrade="false"
+		;;
+	2)
+		# Upgrade
+		upgrade="true"
+		;;
+	*)
+		echo "[ERROR] post-install script called with unknown argument: '$1'"
+		exit 1
+		;;
+esac
+
+chmod 0644 /etc/graylog/graylog_illuminate_core/*
+chmod 0644 /etc/graylog/graylog_illuminate_windows/*
+chmod 0644 /etc/graylog/graylog_illuminate_okta/*
+chmod 0644 /etc/graylog/graylog_illuminate_paloalto/*
+chmod 0644 /etc/graylog/graylog_illuminate_o365/*
+
+if [ ! -f "/etc/graylog/graylog_illuminate_core/static_accounts.csv" ]; then
+	mv /etc/graylog/graylog_illuminate_core/static_accounts.csv.default /etc/graylog/graylog_illuminate_core/static_accounts.csv
```

Comment:
I think we should copy the .csv files instead of renaming them. Once we correctly register them via `config_files` (see below), we should not remove them. They will be removed once the package gets removed.

Keeping the originals around also helps with finding differences with the current version.

Reply:
Okay, good call.